### PR TITLE
cleanup processed files in VR

### PR DESCRIPTION
**Pull request by DaanHoogland**
### Description
When a PR is long running, the directory /var/cache/cloud/processed filles up and might glock the filesystem, rendring the VR unusable. 
This PR implements a mech for cleaning that directory on demand.
(to be triggered yet, by API/schedule/configurable timer???)

### Types of changes

- [ ] Breaking change (fix or feature that would cause existing functionality to change)
- [ ] New feature (non-breaking change which adds functionality)
- [ ] Bug fix (non-breaking change which fixes an issue)
- [x] Enhancement (improves an existing feature and functionality)
- [ ] Cleanup (Code refactoring and cleanup, that may add test cases)

#### Feature/Enhancement Scale

- [ ] Major
- [x] Minor

#### Bug Severity

- [ ] BLOCKER
- [ ] Critical
- [x] Major
- [ ] Minor
- [ ] Trivial

### CODE REVIEW

**Comment by DaanHoogland** [comment]
@blueorangutan package

**Comment by DaanHoogland** [comment]
@blueorangutan package

**Comment by DaanHoogland** [comment]
@blueorangutan test matrix keepEnv

**Comment by DaanHoogland** [comment]
@blueorangutan test keepEnv

File: core/src/main/java/com/cloud/agent/api/CleanProcessedCacheCommand.java [hunk 1]
```diff
@@ -0,0 +1,41 @@
...
+// KIND, either express or implied.  See the License for the
+// specific language governing permissions and limitations
+// under the License.
+//
+
+package com.cloud.agent.api;
+
+import com.cloud.agent.api.routing.NetworkElementCommand;
+
+public class CleanProcessedCacheCommand extends NetworkElementCommand {
```
Comment (DaanHoogland) [inline core/src/main/java/com/cloud/agent/api/CleanProcessedCacheCommand.java:24]: nts: does this mean the answer should also inherit (from NetworkElementAnswer for instance)

**Comment by DaanHoogland** [comment]
@blueorangutan test keepEnv

**Comment by DaanHoogland** [comment]
@blueorangutan package

**Comment by DaanHoogland** [comment]
@blueorangutan test keepEnv

**Comment by DaanHoogland** [comment]
vpc creation failes after these changes: needs work

**Comment by weizhouapache** [comment]
@DaanHoogland 
there was a commit for similar issue see #5331
the script runs once every day. we could let it run every hour or every minute.

**Comment by DaanHoogland** [comment]
> @DaanHoogland there was a commit for similar issue see #5331 the script runs once every day. we could let it run every hour or every minute.

let's push this forward then (marking 4.18.2 for now and we can discuss if it is still applicable)

**Comment by DaanHoogland** [comment]
@blueorangutan LLpackage

**Comment by DaanHoogland** [comment]
@blueorangutan LLtest matrix

**Pull request closed without merging**